### PR TITLE
Support for Lexus NX 300h

### PR DESCRIPTION
**Pull request by kengggg**

### CODE REVIEW

**Comment by kengggg** [comment]
Should I test for both DSU-plugged and unplugged?

**Comment by kengggg** [comment]
Hm. It seems my segment could not be found hence the test_car_model fails. Did I do anything wrong? @vanillagorillaa

File: selfdrive/test/test_car_models.py [hunk 1]
```diff
@@ -313,6 +313,15 @@ def get_route_logs(route_name):
     'carFingerprint': TOYOTA.LEXUS_RXH,
     'enableCamera': True,
     'enableDsu': True,
+  },
```
Comment (pd0wm) [inline selfdrive/test/test_car_models.py]: Think something went wrong solving the merge conflict here. You're adding the RX_TSS2 again, which was added in another PR.

Comment (kengggg) [reply selfdrive/test/test_car_models.py]: I was confused while rebasing, sorry.

**Comment by kengggg** [comment]
This PR is a mess now. Will start over.

**Pull request closed without merging**